src/eko/evolution_operator/__init__.py

```diff
@@ -195,9 +195,9 @@ def quad_ker(
         )
         # scale var expanded is applied on the kernel
         if sv_mode == sv.Modes.expanded and not is_threshold:
-            ker = np.ascontiguousarray(ker) @ np.ascontiguousarray(
+            ker = np.ascontiguousarray(
                 sv.expanded.singlet_variation(gamma_singlet, a1, order, nf, L)
-            )
+            ) @ np.ascontiguousarray(ker)
```

Comment:
uhh well spotted! Did this solved the issue?

Reply:
Unfortunately no ... still I believe this error was mathematically at the same order order as the scale difference, just in practice with the actual numbers it is tiny